### DISCUSSION TITLE
电报是必须有管理同意才给进入吗

### DISCUSSION THREAD

**Discussion by tnnevol** [AlistGo/alist#7264] [Q&A] [answered]
电报一直禁言状态，没办法验证呀

**Comment by Simonzhang8** [comment]
你是不是没及时验证而被禁言了？

**Reply by tnnevol** [reply]
> 你是不是没及时验证而被禁言了？

我一进去就是禁言状态，然后给机器人发完贴图，到群里还是禁言的，机器人给的贴图也没办法发送到群里

**Comment by tnnevol** [comment, marked as answer]
不要使用桌面端应用验证，换成移动端应用，就可以看到群里发送贴纸的入口了，在群里发送一张贴纸就好了
